src/REPLICA/fix_pimd_langevin.h

```diff
@@ -194,6 +196,30 @@ class FixPIMDLangevin : public Fix {
   int size_restart_global();
   int pack_restart_data(double *list);
   void restart(char *buf) override;
+
+   // Planck's constant for each LAMMPS unit style
```

Comment:
These two maps should not be required to be in the header. Instead they could be moved to the .cpp file into an anonymous namespace.